### PR TITLE
[HOTFIX] Bootstrap structure and uploading project in heroku

### PR DESCRIPTION
**Pull request by bendeloyola**
Fixing folder structure bootstrap because it can't compile assets when uploading in heroku.

https://tradance-app.herokuapp.com/

### CODE REVIEW

**Review by neroneronomi — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -39,5 +39,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
-
+master.key
```
Comment (analyn-cajocson) [inline .gitignore]: this is already added

image

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (bendeloyola) [reply .gitignore]: removed another master.key

File: app/views/dashboard/_dashboard_trader.html.erb [hunk 2]
```diff
@@ -1 +1,127 @@
...
+    -moz-user-select: none;
+    user-select: none;
+  }
+
+  @media (min-width: 768px) {
+    .bd-placeholder-img-lg {
+        font-size: 3.5rem;
+    }
+  }
+</style>
```
Comment (analyn-cajocson) [inline app/views/dashboard/_dashboard_trader.html.erb]: can you move these styles to the assets folder?

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]
Add screenshots also since you have some changes in views

**Review by bendeloyola — COMMENTED** [review]
dashboard trader UI will handle by oliver

File: app/views/dashboard/_dashboard_trader.html.erb (hunk 2 above)
Comment (bendeloyola) [reply app/views/dashboard/_dashboard_trader.html.erb]: removed UI dashboard trader. UI will handled by oliver

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**